integration-cli/docker_cli_run_test.go

```diff
@@ -2402,30 +2402,6 @@ func (s *DockerSuite) TestRunExposePort(c *check.C) {
 	c.Assert(out, checker.Contains, "invalid range format for --expose")
 }
 
-func (s *DockerSuite) TestRunUnknownCommand(c *check.C) {
-	out, _, _ := dockerCmdWithStdoutStderr(c, "create", "busybox", "/bin/nada")
```

Comment:
won't work because lookups have to happen inside the container's mount ns after all volumes are mounted because PATH and binaries could live inside them